conftest.py

```diff
@@ -5,15 +5,45 @@
 the proper patching.
 """
 
+import random
 from functools import partial
 from pathlib import Path
 from urllib.parse import urljoin
 
 import pytest
+from django.apps import apps
 from django.contrib.auth import get_user_model
+from django.db import connection
 from django.test import override_settings
 
 
+@pytest.fixture(scope="session")
+def django_db_setup(django_db_setup, django_db_blocker):
+    """Force db sequences to use random numbers."""
+    with django_db_blocker.unblock():
+        randomize_db_sequences()
+
+
+def randomize_db_sequences():
```

Comment:
How would you feel about moving this outside of `conftest` and making a small Django management command wrapper for it? I would love for our initial setup instructions to include randomizing the sequences immediately after database creation because it's not just our tests that suffer from ugly ID synchronization.